### ISSUE TITLE
Warning message is displaying security groups' memory address, instead of name/ID

### ISSUE THREAD

**Issue by ttres** [aws/amazon-vpc-cni-k8s#1922]
IPAM log file is displaying memory addresses instead of names:
```
{"level":"warn","ts":"2022-03-04T14:11:35.782Z","caller":"awsutils/awsutils.go:730",
"msg":"No custom networking security group found, will use the node's primary ENI's SG:
[%!s(*string=0xc000027830) %!s(*string=0xc000027840) %!s(*string=0xc000027850)]"}
```

Error text points to this code:
https://github.com/aws/amazon-vpc-cni-k8s/blob/828385ba872702c2639dedc0e5f5faf006956e27/pkg/awsutils/awsutils.go#L822

**Comment by jayanthvn** [comment]
Hi, I recently fixed this with #1892. PR is merged to master.

**Comment by ttres** [comment]
Thanks! Closing.
